safe-global/safe-wallet-web
# feat: add WalletConnect v2 wallet module

## What it solves

Partially resolves #2017

## How this PR fixes it

A second (v2) WalletConnect module has been added to onboard. It can be toggled via the config service with wallet name `walletConnect_v2` and has been enabled on all non-prod. chains.

Note: onboard dedupes wallet modules and the label of the old module has therefore been modified from "WalletConnect" to "WalletConnect v1".

## How to test it

Connect a wallet and observe the "WalletConnect v1" and "WalletConnect" modules are present in the modal.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/8b01742e-a5bc-4af2-9aa9-9e9b0c2bf39a)

## Checklist
